rclpy/src/rclpy/_rclpy.c

```diff
@@ -5093,10 +5094,19 @@ PyMODINIT_FUNC PyInit__rclpy(void)
     return NULL;
   }
 
+  RCLError = PyErr_NewExceptionWithDoc(
+    "_rclpy.RCLError",
+    "Thrown when there is an error in rcl.",
+    PyExc_RuntimeError, NULL);
+  if (PyModule_AddObject(m, "RCLError", RCLError)) {
```

Comment:
```suggestion
  if (PyModule_AddObject(m, "RCLError", RCLError) != 0) {
```

For consistency with the other errors.